examples/06-transaction.ts

```diff
@@ -1,11 +1,12 @@
-import { DataUtils, graphqlEndpoints } from '../src';
+import { DataUtils } from '../src';
 import * as dotenv from 'dotenv';
 
 dotenv.config();
+const dataApiKey = 'eyJvcmciOiI2NTIzZjY5MzUwOTBmNzAwMDFiYjJkZWIiLCJpZCI6IjI4ZWJiMGQ5YTMxYjQ3MmY4NmU4MWY2YTVhYzBhMzE1IiwiaCI6Im11cm11cjEyOCJ9';
```

Comment:
I would move this into DataUtils and use it as the default in case no key is passed.

Reply:
Done